src/components/AccountForms/RegistrationForm/ChipComponent.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from 'react';
+import useStyles from './RegistrationForm.styles';
+import './registration.css';
+import { InputLabel, Chip } from '@material-ui/core';
+import CheckCircleOutlineIcon from '@material-ui/icons/CheckCircleOutline';
+
+const ChipComponent = ({ register, unregister, watch, setValue, errors }) => {
+  const classes = useStyles();
+  const watchRole = watch('role_id', '');
+  const handleClick = (event) => {
+    setValue('role_id', 1, {
```

Comment:
to make this reusable, change the 1 to use the id from the event object instead.
you might have to ensure that event.target.id is an integer instead of str

if you'd like, you can also change the hard coded 'role_id' to use the name as well
double check if this works as intended

```
setValue(event.target.name, parseInt(event.target.id), {
    ...
}

Reply:
@PeaWarrior I've got it working now. The only issue is the styling changed, so I'm not sure yet how to fix that.

Reply:
> @PeaWarrior I've got it working now. The only issue is the styling changed, so I'm not sure yet how to fix that.

Nice. I don't think that should change any of the styling. Might be something else